src/components/searchResults.tsx

```diff
@@ -0,0 +1,55 @@
+
+import { AccountDisplay } from '@src/components/accountDisplay';
+import { SearchResultsItem } from '@src/components/searchResultsItem';
+import { Account, AccountFields} from '@src/generated/sobs';
+import { Form } from 'antd';
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+
+interface SearchResultsProps {
+    accs: any;
+}
+
+interface SearchResultsStates {
+    selectedAccount: AccountFields;
+}
+
+export class SearchResults extends React.Component<SearchResultsProps, SearchResultsStates> {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedAccount: null,
+        };
+    }
+
+    public setSelectedAccount = (account: AccountFields) => {
+        this.setState({selectedAccount: account});
+    }
+
+    public render() {
+        if (!this.props.accs) {
+          return <div>No Accounts Found</div>;
+        }
+
+        if (this.state.selectedAccount) {
```

Comment:
I don't think it makes sense for the Search Results component to be responsible for rendering the selected account.  

1: this just doesn't really fit into the component hierarchy.  EG: it wouldn't really make sense to render the account here as then the search controls would still be show.  

2: It add additional dependancies.  Search Results should only be responsible for displaying the results... not displaying the selected account.  Once an account is "Selected" the interface should just replace the search components with the `AccountDisplay` component.